Gemfile

```diff
@@ -5,9 +5,6 @@ ruby "2.3.1"
 gem 'rails', '4.2.5'
 gem 'rails-api'
 gem 'spring', :group => :development
-gem 'omniauth-oauth2', '~> 1.3.1'
-gem 'omniauth-slack'
-gem "omniauth-google-oauth2"
```

Comment:
I thought we planned to have the normal authentication in place, in case Andela Auth isn't an option